Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -3361,6 +3361,7 @@ enum class WebCore::FetchOptionsDestination : uint8_t {
     Audioworklet,
     Document,
     Embed,
+    Environmentmap,
```

Comment:
this should probably be `EnvironmentMap` with a capital-M, I'm not sure why the other values in the enum aren't camel cased like the other enums in this file

Reply:
I'd keep it as is for consistency sake but I agree with you.